crest/Assets/Crest/Crest/Scripts/Helpers/UnderwaterEffect.cs

```diff
@@ -59,12 +61,13 @@ private void Start()
 #endif
             _rend = GetComponent<Renderer>();
 
+            _camera = transform.parent.gameObject.GetComponent<Camera>();
```

Comment:
should this be both in the inspector (its serialised above), and also set from code? i'd expect one of or the other, or setting from code only if its not populated by the user.

it might be better to use GetComponentInParent, although it can work like this too

Reply:
I have removed serialisation as validation already enforces parent camera relationship. Also now using GetComponentInParent. Thanks.